claim_batch/services.py

```diff
@@ -532,6 +542,94 @@ def do_process_batch(audit_user_id, location_id, period, year):
         logger.info("no product found in  %s for %s/%s", location_id, period, year)
 
 
+# Calculate allcated contributions
+def get_allocated_premium(premiums, start_date, end_date):
+    # go trough the contribution and find the allocated contribution 
+    allocated_premiums = 0
+    for premiums in premium:
+        allocation_start = max(policy.effective_date, start_date)
+        allocation_stop = min(start_date, policy.expiry_date)
+        policy_duration = policy.expiry_date(  - policy.effective_date 
```

Comment:
@delcroip Is this done om purpose? 'policy_duration = policy.expiry_date(  - policy.effective_date '